hadoop-ozone/dist/src/main/compose/ozones3-haproxy/test.sh

```diff
@@ -18,12 +18,14 @@
 COMPOSE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 export COMPOSE_DIR
 
+export SECURITY_ENABLED=false
```

Comment:
We do run robot tests for S3 in a secure environment. We should file a jira or do the work to include HTTPS and secure S3 tests for sample HA Proxy and testing of HA Proxy.

Reply:
Each environment is either secure or not (see `ozone` vs. `ozonesecure`).  This one is not, which is not changed by this PR.  The `SECURITY_ENABLED` variable is required for the S3 tests to decide whether to run the secure-specific commands.  Please file a Jira if you would like to add the security-related containers and settings.

Reply:
Yup, I saw that. The goal of this change is to test the HA Proxy setup with S3G, and most of the exciting bits are related to security. I will file a Jira...

Reply:
https://issues.apache.org/jira/browse/HDDS-7990